app/models/feed.rb

```diff
@@ -10,9 +10,7 @@ def get(limit, max_id = nil, since_id = nil)
     max_id     = '+inf' if max_id.blank?
     since_id   = '-inf' if since_id.blank?
     unhydrated = redis.zrevrangebyscore(key, "(#{max_id}", "(#{since_id}", limit: [0, limit], with_scores: true).map(&:last).map(&:to_i)
-    status_map = Status.where(id: unhydrated).cache_ids.map { |s| [s.id, s] }.to_h
-
-    unhydrated.map { |id| status_map[id] }.compact
+    Status.where(id: unhydrated).cache_ids
```

Comment:
Ah! I was wondering why this was necessary. Thanks for the explanation. I've updated the test to reflect the requirement of maintaining the order from Redis. They now fail with my "refactor" but pass with the original implementation 👍